src/styles/MessageInputFlat.scss

```diff
@@ -81,6 +86,15 @@
           }
         }
       }
+
+      .str-chat__message-attachment-card {
```

Comment:
these affect React SDK as well: fixes quoted message preview in message input for card attachments (like GIF or link)

Reply:
great catch!